commit 82bb5d326e59b02c75a01429776b931246b53a03
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-29T13:46:42+00:00 (2 months ago)
    
    TOOLS-2299 bin/reflash needs to regenerate build.spec.merged
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/bin/reflash b/bin/reflash
index b05fe881..ece66249 100755
--- a/bin/reflash
+++ b/bin/reflash
@@ -179,6 +179,12 @@ check_prerequisites()
     else
         fatal "Unsupported platform ${uname}"
     fi
+
+    # make sure we're using an up to date build spec
+    (cd $ROOT && gmake --quiet build.spec.merged) || {
+        echo "failed to update $ROOT/build.spec.merged" 2>&1
+        exit 1
+    }
 }
 
 check_user_sanity()
